tests/python/gaia-ui-tests/gaiatest/apps/system/regions/activities.py

```diff
@@ -30,35 +30,39 @@ def __init__(self, marionette):
 
     def tap_wallpaper(self):
         self.marionette.find_element(*self._wallpaper_button_locator).tap()
-        self.wait_for_element_not_displayed(*self._actions_menu_locator)
+        Wait(self.marionette).until(
+            expected.element_not_displayed(*self._actions_menu_locator))
```

Comment:
If this element is visible before we tap the wallpaper button then I would recommend finding it before the tap. This will allow the test to fail if the element locator changes.